internal/k8s/crdcontroller.go

```diff
@@ -31,40 +31,42 @@ import (
 	akov1alpha2 "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/apis/ako/v1alpha2"
-	akocrd "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/client/v1alpha1/clientset/versioned"
-	akoinformers "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/client/v1alpha1/informers/externalversions"
+	akov1beta1 "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/apis/ako/v1beta1"
+	v1beta1akocrd "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/client/v1beta1/clientset/versioned"
+
 	v1alpha2akoinformers "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/client/v1alpha2/informers/externalversions"
+	v1beta1akoinformers "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/client/v1beta1/informers/externalversions"
 
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
 
 	"k8s.io/client-go/tools/cache"
 )
 
-func NewCRDInformers(cs akocrd.Interface) {
-	var akoInformerFactory akoinformers.SharedInformerFactory
-
-	akoInformerFactory = akoinformers.NewSharedInformerFactoryWithOptions(cs, time.Second*30)
-	hostRuleInformer := akoInformerFactory.Ako().V1alpha1().HostRules()
-	httpRuleInformer := akoInformerFactory.Ako().V1alpha1().HTTPRules()
-	aviSettingsInformer := akoInformerFactory.Ako().V1alpha1().AviInfraSettings()
-
+func NewCRDInformers() {
 	v1alpha2akoInformerFactory := v1alpha2akoinformers.NewSharedInformerFactoryWithOptions(
 		lib.AKOControlConfig().V1alpha2CRDClientset(), time.Second*30)
 	ssoRuleInformer := v1alpha2akoInformerFactory.Ako().V1alpha2().SSORules()
 	l4RuleInformer := v1alpha2akoInformerFactory.Ako().V1alpha2().L4Rules()
 
+	//v1beta1 informer initialization
+	v1beta1akoInformerFactory := v1beta1akoinformers.NewSharedInformerFactoryWithOptions(
+		lib.AKOControlConfig().V1beta1CRDClientset(), time.Second*30)
+	aviInfraSettingInformer := v1beta1akoInformerFactory.Ako().V1beta1().AviInfraSettings()
+	hostRuleInformer := v1beta1akoInformerFactory.Ako().V1beta1().HostRules()
+	httpRuleInformer := v1beta1akoInformerFactory.Ako().V1beta1().HTTPRules()
+
 	lib.AKOControlConfig().SetCRDInformers(&lib.AKOCrdInformers{
 		HostRuleInformer:        hostRuleInformer,
 		HTTPRuleInformer:        httpRuleInformer,
-		AviInfraSettingInformer: aviSettingsInformer,
 		SSORuleInformer:         ssoRuleInformer,
 		L4RuleInformer:          l4RuleInformer,
+		AviInfraSettingInformer: aviInfraSettingInformer,
 	})
 }
 
-func NewInfraSettingCRDInformer(cs akocrd.Interface) {
-	akoInformerFactory := akoinformers.NewSharedInformerFactoryWithOptions(cs, time.Second*30)
-	aviSettingsInformer := akoInformerFactory.Ako().V1alpha1().AviInfraSettings()
+func NewInfraSettingCRDInformer(cs v1beta1akocrd.Interface) {
+	akoInformerFactory := v1beta1akoinformers.NewSharedInformerFactoryWithOptions(cs, time.Second*30)
```

Comment:
`cs` can be replaced with `lib.AKOControlConfig().V1beta1CRDClientset()` similar to the change you have done in `NewCRDInformers` function.